csi/src/block_vol.rs

```diff
@@ -235,7 +235,7 @@ fn findmnt_device(mountpoint: &str) -> Result<Option<String>, DeviceError> {
 fn equals_findmnt_device(findmnt_device_path: &str, device_path: &str) -> bool {
     lazy_static! {
         static ref RE_FINDMNT: regex::Regex = regex::Regex::new(
-            r"(?x).*dev\[(?P<device>/.*)\]
+            r"(?x).*\[(?P<device>/.*)\]
```

Comment:
The code comment for this function should be expanded to document what it's trying to match.

Reply:
Done https://github.com/openebs/Mayastor/pull/407/commits/5e8a615aeb84545604a1d443c02cb6c0e08a3162